core/src/main/resources/hudson/cli/Messages.properties

```diff
@@ -113,3 +113,14 @@ WaitNodeOfflineCommand.ShortDescription=Wait for a node to become offline.
 
 CliProtocol.displayName=Jenkins CLI Protocol/1 (unencrypted)
 CliProtocol2.displayName=Jenkins CLI Protocol/2 (transport encryption)
+
+LegacyWrapperCLIReturnCode.UNSPECIFIED_REASON=Unspecified reason
+
+StandardCLIReturnCode.OK=Everything went OK
```

Comment:
Did you not mean to use this?

Reply:
I used until I thought about the channel issue discussed above. So it needs either to be removed (as unused) or put back in place.